src/rprocessing/applet/BuiltinApplet.java

```diff
@@ -199,12 +206,54 @@ public void stroke(double v1, double v2, double v3, double alpha) {
         super.stroke((float) v1, (float) v2, (float) v3, (float) alpha);
     }
 
+    /*
+     * 3D
+     */
+
+    public void box(double size) {
+        super.box((float) size);
+    }
+
+    public void box(double w, double h, double d) {
+        super.box((float) w, (float) h, (float) d);
+    }
+
     // TODO: Blocked until pre-processor.
```

Comment:
Good documentation of ongoing issues. Color issues you encounter in this project will often take two broad forms:

1. the `color` data type in processing is actually pre-processor sugar for an int
2. specifics of the way colors are handled at a low level using ARGB bit-math, vs. control of how colors are specified or retrieved using colorMode().